packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -55,10 +55,12 @@ final Platform macPlatform = FakePlatform(
 void main() {
   MockFlutterVersion mockFlutterVersion;
   BufferLogger logger;
+  FakeProcessManager fakeProcessManager;
```

Comment:
nit: as we do these, should we start adding NNBD annotations?  e.g.
```suggestion
  /*late*/ FakeProcessManager fakeProcessManager;
```

Reply:
I think we probably need to attack the migration a little more systematically, and I'd worry that a smattering of partial migrations might confuse things a bit.

Reply:
Yeah, these will get out of sync too.